doc/readthedocs/tut_solid_preexo.rst

```diff
@@ -3,14 +3,192 @@
+we choose the model ``MAT_Struct_Struct_PlasticNlnLogNeoHooke`` to accomplish this task.
+
+The geometry, finite element mesh and node sets for the boundary conditions are created using Cubit.
+The mesh is rather fine and 27-node brick elements have been used herein.
+The corresponding journal file can be run to reproduce this mesh and output a binary exodus mesh information file ``tutorial_solid.e``.
+The geometry dimensions (actually mainly the radius, all other dimensions depend on it) and element size can be varied, since they are parametrized in the journal file.
+
+.. literalinclude::  tutorial_solid.jou
+   :linenos:
+
+The general definition of the geometry in this cubit journal file, which can be run from the directory ``<source-root>/tests/framework-test/`` includes the complete mesh and four surface node sets.
+If you don't have Coreform Cubit, you may use the resulting exodus file directly, which is also located in this directory.
+
+The boundary conditions and loading, the material and all solver details are defined in two other files:
+A so-called boundary condition file ``tutorial_solid.bc`` and a header file ``tutorial_solid.head``.
+These three files together can be combined to a |FOURC| input file using the command
+
+::
+
+    <build-dir>/pre-exodus --exo=tutorial_solid.e --bc=tutorial_solid.bc --head=tutorial_solid.head
+
+Note that you need to build the ``pre_exodus`` command before.
+You'll find the build information for ``pre_exodus`` in the :ref:`setup section <custom_target_specifiers>` or in the :ref:`workflow/preprocessing section <preprocessing>`.
+
+The boundary condition file ``tutorial_solid.bc`` contains the element information and boundary conditions.
+The element type is a quadratic (27 node) 3D solid element(Solid HEX27) with nonlinear kinematics, that is, including a large strain formulation.
+The boundary conditions include three symmetry conditions (based on the surfaces defined in Cubit) and one loading condition.
+Note that only surface conditions are given; the lines that belong to two surfaces automatically get the combined boundary conditions.
+The loading condition has a function number (1) included for the y-direction, and also gets a tag: ``TAG monitor_reaction``.
+The tag is for an extra output of the total forces, which are printed in a csv file, if the output is requested.
+
+.. literalinclude:: tutorial_solid.bc
+   :linenos:
 
-.. note::
 
-   Here I would appreciate having a tutorial for solid mechanics as well.
-   Jan Schnabel did a nice solid mechanics introduction that might be suitable.
+
+
+All control parameters for the material, solvers, loading function, output information are contained in ``tutorial_solid.head``.
+Particularly, the output of the forces is requested by a particular section, ``--IO/MONITOR STRUCTURE DBC``, which controls accuracy and frequency of the output.
```

Comment:
```suggestion
Particularly, the output of the forces is requested by a particular section, ``IO/MONITOR STRUCTURE DBC``, which controls accuracy and frequency of the output.
```

Let's stop prefixing sections with `--`. Even for dat, this is kind of an implementation detail; for yaml/json, it is just wrong.